### PR TITLE
add Joint_states_subscriber to apps

### PR DESCRIPTION
**Pull request by Needrom**
add joint_states topic for test

### CODE REVIEW

File: apps/joint_states_subscriber/app.c [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+#define ARRAY_LEN 200
+#define JOINT_DOUBLE_LEN 20
+
+#define RCCHECK(fn) { rcl_ret_t temp_rc = fn; if((temp_rc != RCL_RET_OK)){printf("Failed status on line %d: %d. Aborting.\n",__LINE__,(int)temp_rc); return 1;}}
+#define RCSOFTCHECK(fn) { rcl_ret_t temp_rc = fn; if((temp_rc != RCL_RET_OK)){printf("Failed status on line %d: %d. Continuing.\n",__LINE__,(int)temp_rc);}}
+
+rcl_subscription_t joint_states_subscriber;
+sensor_msgs__msg__JointState joint_states_msg;
+
+char test_array[ARRAY_LEN];
```
Comment (pablogs9) [inline apps/joint_states_subscriber/app.c:20]: Delete

File: apps/joint_states_subscriber/app.c [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+void subscription_joint_state_callback(const void *msgin){
+	const sensor_msgs__msg__JointState *msg = (const sensor_msgs__msg__JointState *)msgin;
+	printf("I get joint_state topic msg.pos: %d , %x , %f \r\n", (int)(msg->position.data[6] * 100), msg->position.data[6], msg->position.data[6]);
+}
+
+//int appMain(int argc, const char * const * argv)
+int appMain(void *argument)
+{
+  	memset(test_array,'z',ARRAY_LEN);
```
Comment (pablogs9) [inline apps/joint_states_subscriber/app.c:30]: Delete

**Review by pablogs9 — CHANGES_REQUESTED** [review]

**Comment by Needrom** [comment]
OK

**Comment by pablogs9** [comment]
For the next time, if you push commits to the same branch the PR is updated. You don't need to close and open a new one.

**Pull request closed without merging**